hangupsbot/plugins/_chatbridge/chatbridge_syncrooms.py

```diff
@@ -59,25 +59,55 @@ def _send_to_external_chat(self, config, event):
         message = event.passthru["original_request"]["message"]
         image_id = event.passthru["original_request"]["image_id"]
 
+        is_action = event.passthru["chatbridge"].get("source_action")
+
         if not message:
             message = ""
 
+        if any(a.type == 4 for a in event.conv_event._event.chat_message.annotation):
```

Comment:
fragile, the event can also be a bridge `FakeEvent`, which does not contain `conv_event`. this fails on a slack <> ho <> slack combination, stopping any other relays in sequence. the error when a message is sent on a slackrtm-synced channel is:
```
13:18:27 ERROR handlers: sending: plugins._chatbridge.chatbridge_syncrooms._broadcast : coroutine
Traceback (most recent call last):
  File "/root/projects/hangoutsbot/hangupsbot/handlers.py", line 350, in run_pluggable_omnibus
    yield from function(*_passed)
  File "/root/projects/hangoutsbot/hangupsbot/webbridge/__init__.py", line 165, in _broadcast
    passthru = passthru ))
  File "/root/projects/hangoutsbot/hangupsbot/plugins/_chatbridge/chatbridge_syncrooms.py", line 67, in _send_to_external_chat
    if any(a.type == 4 for a in event.conv_event._event.chat_message.annotation):
AttributeError: 'FakeEvent' object has no attribute 'conv_event'
```